features/networking/multus.feature

```diff
@@ -1263,3 +1263,49 @@ Feature: Multus-CNI related scenarios
       | /usr/sbin/ip | route         |
     Then the output should contain:
       | 192.168.10.0                 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-30054
+  @admin
+  Scenario: Multus namespaceIsolation should allow references to CRD in the default namespace
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    # Create the net-attach-def via cluster admin
+    Given I switch to cluster admin pseudo user
+    And I use the "default" project
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-macvlan.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %>  |     
+      | ["spec"]["config"]|'{ "cniVersion": "0.3.0", "type": "macvlan", "mode": "bridge", "ipam": { "type": "whereabouts", "range": "192.168.22.100/24"} }'|
+    Then the step should succeed
+
+    #Cleanup created net-attach-def from default namespaces
+    Given I register clean-up steps:
+    """
+    I run the :delete admin command with:
+      | object_type       | net-attach-def             |
+      | object_name_or_id | macvlan-bridge-whereabouts |
+    the step should succeed
+    """
+    
+    # Create a pod absorbing above net-attach-def
+    Given I switch to the first user
+    And I create a new project
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/Pods/generic_multus_pod.yaml" replacing paths:
+      | ["metadata"]["name"] | macvlan-bridge-whereabouts-pod1                                           |
```

Comment:
Indentation

Reply:
@anuragthehatter  Should Indentation in line 1294 be line up with 1295 and 1296? Thanks!

Reply:
Example : https://github.com/openshift/verification-tests/blob/ac9de1200866746f9ecd32c04c9574b6feaa48fa/features/networking/multus.feature#L1247

Reply:
@anuragthehatter Fixed indentation issue, thanks!

Reply:
@weliang1 Thanks! Can you also do the same on 1277-1278?

Reply:
@anuragthehatter Fixed it, thanks!

Reply:
Sorry , you might have to take a look at 1277 again :)

Reply:
@anuragthehatter Should fix it this time, thanks!